### ISSUE TITLE
Rules `Indentation` and `FunctionSignature` can't agree on the correct format when arguments are multiline with `indentStyle = tabs`

### ISSUE THREAD

**Issue by cyyynthia** [detekt/detekt#9295]
## Expected Behaviour
The following code should yield no lint issues, or at least stabilise into a format all rules agree with.

```kt
class A {
	fun func(
		arg1: SomeLongType,
		arg2: SomeOtherLongType,
		arg3: AnotherLongTypeNameBecauseWeLoveJavaNamingSchemeFactory,
		arg4: RunningOutOfTypeNameIdeas,
	) {
		/* ... */
	}
}
```

## Observed Behaviour
Both rules `Indentation` and `FunctionSignature` conflict.
- `Indentation` wants only tabs to indent, including as the 2nd indent within the arguments list
- `FunctionSignature` transforms the 2nd indentation level into 4 spaces instead of a tab

## Steps to Reproduce
Try to lint the above snippet with Detekt.

## Context
Tab support was introduced in the current release, but it seems to have some edge cases unaccounted for at this time.

## Your Environment

* Version of detekt used: 2.0.0-alpha.3
* Version of Gradle used (if applicable): 9.4.1
* Gradle scan link (add `--scan` option when running the gradle task): N/A
* Operating System and version: Windows 11
* Link to your project (if it's a public repository): private

**Comment by ChristophyBarth** [comment]
Hi, I'd like to pick this up. I will be starting on it now and will open a PR once I have something working.

**Comment by cyyynthia** [comment]
I'm still experiencing this issue on detekt 2-alpha.5

**Comment by ChristophyBarth** [comment]
> I'm still experiencing this issue on detekt 2-alpha.5

Interesting, let me revisit this 🤔

**Comment by ChristophyBarth** [comment]
Hey @cyyynthia, the fix from #9315 is actually fine; the problem is the config key. I renamed it (based on a request) to `indentStyle` (camelCase) during the PR's review, but the changelog still says `indent_style`, and detekt silently ignores unknown keys on plugin rule sets, so `indent_style: tab` does nothing, and both rules just fall back to spaces.

This one actually works:
```yaml
ktlint:
   indentStyle: tab
   Indentation:
      active: true
   FunctionSignature:
      active: true
```

(the value is `tab`, not `tabs`). With that, your snippet lints clean for me. I'll update the misleading changelog either way. Could you confirm the key sorts it out on your end?

**Comment by cyyynthia** [comment]
Ah I had the configuration in `Indentation` still instead of directly within `ktlint`. Changing that does the trick, although it seems another side effect of using tabs is that it doesn't count a tab as multiple characters, so line length "increases" by 3 - but that's a different issue.
